packages/nc-gui-v2/components/smartsheet-column/LinkedToAnotherRecordOptions.vue

```diff
@@ -0,0 +1,109 @@
+<script setup lang="ts">
+import type { TableType } from 'nocodb-sdk'
+import { ModelTypes, MssqlUi, SqliteUi } from 'nocodb-sdk'
+import { useColumnCreateStoreOrThrow } from '#imports'
+import { MetaInj } from '~/context'
+import MdiPlusIcon from '~icons/mdi/plus-circle-outline'
+import MdiMinusIcon from '~icons/mdi/minus-circle-outline'
+
+const { formState, validateInfos, onDataTypeChange, setAdditionalValidations } = $(useColumnCreateStoreOrThrow())
+const { tables, sqlUi } = $(useProject())
+const meta: TableType = $(inject(MetaInj))
+
+console.log(tables)
+
+setAdditionalValidations({
+  childId: [{ required: true, message: 'Required' }],
```

Comment:
this should return promise.

Example:

```js
const validators = {
  'childId': [
    {
      validator: (_: any, value: any) => {
        return new Promise<void>((resolve, reject) => {
          if (!checkSomething(value)) {
            return reject(new Error('Some error messages'))
          }
          resolve()
        })
      },
    },
  ],
}
```

```js
setAdditionalValidations({
  ...validators,
})
```

Reply:
@wingkwong The validator here is simple so took this approach